library/elliptics.h

```diff
@@ -430,6 +430,7 @@ struct dnet_work_io {
 	int			thread_index;
 	uint64_t		trans;
 	pthread_t		tid;
+	int			joined;
```

Comment:
This is a sign that there is a bug somewhere which allows to call thread cleanup multiple times. It should be fixed without this flag.